.github/workflows/build.yml

```diff
@@ -64,6 +68,13 @@ jobs:
           brew install ccache
           ccache -V
 
+      - name: install ndk
+        if: startsWith(matrix.config.compiler, 'ndk-')
+        run: echo "y" | sudo ${ANDROID_HOME}/cmdline-tools/latest/bin/sdkmanager --install `echo "${{ matrix.config.compiler }}" | tr - ";"`
+      - name: expand ANDROID_HOME env var in conan profile
```

Comment:
Couldn't find a way to put $ANDROID_HOME in conan profile. Don't want to hardcode the path because it would be unusable on my development machine

Reply:
I think we had a similar problem in the android repo. Hardcoding would be last resort but I don't think it is terrible because it is an github specific profile. Locally we could emulate the github ci with https://github.com/nektos/act

Reply:
I should file an enhancement request to conan, because I don't believe we are the only ones who want to put $ANDROID_HOME env var in conan profile